plugin/pkg/auth/authorizer/rbac/bootstrappolicy/policy.go

```diff
@@ -317,7 +317,7 @@ func ClusterRoles() []rbacv1.ClusterRole {
 
 				rbacv1helpers.NewRule(Read...).Groups(autoscalingGroup).Resources("horizontalpodautoscalers").RuleOrDie(),
 
-				rbacv1helpers.NewRule(Read...).Groups(batchGroup).Resources("jobs", "cronjobs").RuleOrDie(),
+				rbacv1helpers.NewRule(Read...).Groups(batchGroup).Resources("jobs", "cronjobs", "jobs/status").RuleOrDie(),
 
 				rbacv1helpers.NewRule(Read...).Groups(extensionsGroup).Resources("daemonsets", "deployments", "deployments/scale",
 					"ingresses", "replicasets", "replicasets/scale", "replicationcontrollers/scale",
```

Comment:
replicasets, statefulsets, daemonsets, deployments, ingresses, horizontalpodautoscalers, and poddisruptionbudgets all have status subresources and aren't included here. If we think status is reasonable to include in this role, we should include those

Reply:
Done, added all of these as well.. A bit confusing that right now `ingresses` is both in `extensions` and `networking` groups.